README.md

```diff
@@ -11,82 +12,87 @@ Logging from the following services is supported for both cases:
 * [RedShift](https://aws.amazon.com/redshift/)
 * [S3](https://aws.amazon.com/s3/)
 
+## Terraform Versions
+
+Terraform 0.12. Pin module version to ~> 5.1.0. Submit pull-requests to master branch.
+
+Terraform 0.11. Pin module version to ~> 3.5.0. Submit pull-requests to terraform011 branch.
+
 ## Usage for a single log bucket storing logs from all services
 
     # Allows all services to log to bucket
-    module "aws_logs" {
+    module "aws\_logs" {
       source         = "trussworks/logs/aws"
-      s3_bucket_name = "my-company-aws-logs"
+      s3\_bucket\_name = "my-company-aws-logs"
       region         = "us-west-2"
     }
 
 ## Usage for a single log bucket storing logs from a single service
 
     #  Allows only the service specified (elb in this case) to log to the bucket
-    module "aws_logs" {
+    module "aws\_logs" {
       source         = "trussworks/logs/aws"
-      s3_bucket_name = "my-company-aws-logs-elb"
+      s3\_bucket\_name = "my-company-aws-logs-elb"
       region         = "us-west-2"
-      default_allow  = false
-      allow_elb      = true
+      default\_allow  = false
+      allow\_elb      = true
```

Comment:
I thought terraform docs 0.8.2 fixed this? Maybe not. Its a little annoying but I won't block on it.

Reply:
Not sure.  I just updated yesterday, so I'd think I was on the latest version.